### PR TITLE
feat: Project-Euler task1 "Multiples of  3 or 5"

### PR DESCRIPTION
**Pull request by adithyanmadhi**
For finding the sum of all the multiples of 3 or 5 below 1000 using Javascript
Screenshot 2024-01-22 230840

Project Euler.net Link: https://projecteuler.net/problem=1
Jira Link: https://mahendrankannan.atlassian.net/browse/ON-17

### CODE REVIEW

File: project-euler/task1/task1.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      let sum = 0;
+
+      for (let i = 1; i < limit; i++) {
+        if (i % 3 === 0 || i % 5 === 0) {
+          sum += i;
+        }
+      }
+
+      return sum;
+    }
```
Comment (gkmahendran09) [inline project-euler/task1/task1.html:23]: We can move this to a separate .js file

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
add a readme file

File: project-euler/task1/task1.html (hunk 1 above)
Comment (adithyanmadhi) [reply project-euler/task1/task1.html]: Created a script.js file seperately and moved the code .

File: project-euler/task1/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Project Euler
```
Comment (gkmahendran09) [inline project-euler/task1/README.md:1]: Markdown syntax to highlight the title for example you could use
```suggestion
# Project Euler
```

File: project-euler/task1/README.md
Comment (gkmahendran09) [inline project-euler/task1/README.md:1]: You could also add the problem URL in this file

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
- Use https://draw.io/ and create a flow chart and upload it in the same folder
- Add one more file for Algorithm

**Comment by gkmahendran09** [comment]
Update the PR title, remove the redundant text `Upload initial...` in all the PRs

File: project-euler/task1/README.md (hunk 2 above)
Comment (adithyanmadhi) [reply project-euler/task1/README.md]: Highlighted in the README.md file.

File: project-euler/task1/README.md
Comment (adithyanmadhi) [reply project-euler/task1/README.md:1]: Added the URL in the README.md file

File: project-euler/task1/algorithm.js
Comment (gkmahendran09) [inline project-euler/task1/algorithm.js:1]: Algorithm's it can be `.txt` file

**Review by gkmahendran09 — APPROVED** [review]

**Pull request merged**